templates/email.html

```diff
@@ -0,0 +1,135 @@
+        padding: 0 !important;
+        width: 100% !important;
+      }
+      @media screen and (max-width: 525px) {
+        .wrapper {
+          width: 100% !important;
+          max-width: 100% !important;
+        }
+        .responsive-table {
+          width: 100% !important;
+        }
+        .padding {
+          padding: 10px 5% 15px 5% !important;
+        }
+        .section-padding {
+          padding: 0 15px 50px 15px !important;
+        }
+      }
+      .form-container {
+        margin-bottom: 24px;
+        padding: 20px;
+        border: 1px dashed #ccc;
+      }
+      .form-heading {
+        color: #2a2a2a;
+        font-family: 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
+        font-weight: 400;
+        text-align: left;
+        line-height: 20px;
+        font-size: 18px;
+        margin: 0 0 8px;
+        padding: 0;
+      }
+      .form-answer {
+        color: #2a2a2a;
+        font-family: 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
+        font-weight: 300;
+        text-align: left;
+        line-height: 20px;
+        font-size: 16px;
+        margin: 0 0 24px;
+        padding: 0;
+      }
+      div[style*='margin: 16px 0;'] {
+        margin: 0 !important;
+      }
+    </style>
+  </head>
+  <body style="margin: 0 !important; padding: 0 !important; background: #fff">
+    <div
+      style="
+        display: none;
+        font-size: 1px;
+        color: #fefefe;
+        line-height: 1px;
+        max-height: 0px;
+        max-width: 0px;
+        opacity: 0;
+        overflow: hidden;
+      "
+    ></div>
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td
+          bgcolor="#ffffff"
+          align="center"
+          style="padding: 10px 15px 30px 15px"
+          class="section-padding"
+        >
+          <table
+            border="0"
+            cellpadding="0"
+            cellspacing="0"
+            width="100%"
+            style="max-width: 500px"
+            class="responsive-table"
+          >
+            <tr>
+              <td>
+                <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                  <tr>
+                    <td>
+                      <table
+                        width="100%"
+                        border="0"
+                        cellspacing="0"
+                        cellpadding="0"
+                      >
+                        <tr>
+                          <td
+                            style="
+                              padding: 0 0 0 0;
+                              font-size: 16px;
+                              line-height: 25px;
+                              color: #232323;
+                            "
+                            class="padding message-content"
+                          >
+                            <h2>Appointment Request Information</h2>
+                            <div class="form-container">${htmlData}</div>
```

Comment:
Body could include a welcoming message as mentioned above. The email will be one of the important services of our product with which the user will interact. Making it little thoughtful will be a great experience.